backend/model/Record.ts

```diff
@@ -0,0 +1,20 @@
+import mongoose, { Schema, model, connect, Types } from 'mongoose';
+
+export interface IRecord extends Document {
+    attendanceDate: string;
+    numberOfCheats: Date;
```

Comment:
`numberOfCheats` refer to the number of times user cheated so it should be type `number`
```suggestion
    numberOfCheats: number;
```